plugins/command_cedict.py

```diff
@@ -0,0 +1,120 @@
+import re
+
+from pyrogram import Client, filters
+from pyrogram.types import Message, \
+    InlineKeyboardMarkup, \
+    InlineKeyboardButton, \
+    CallbackQuery
+
+from tools.cedict import get_cedict_result
+from tools.utils import command_args_to_str
+
+CEDICT_ENTRIES_PER_PAGE = 10
+
+
+@Client.on_message(filters.command(["cedict", "dict", "dic"])
+                   & filters.incoming
+                   & ~filters.edited)
+async def command_cedict(_, message: Message):
+    """/cedict question - get translation from cedict"""
+    query = command_args_to_str(message.command)
+    if query:
+        first_page = 0
+        cedict_response = get_cedict_result(query,
+                                            first_page,
+                                            CEDICT_ENTRIES_PER_PAGE)
+
+        reply_markup = None
+        if cedict_response.has_next_page:
+            reply_markup = _make_next_page_button()
+
+        await message.reply(cedict_response.formatted_result,
+                            reply_markup=reply_markup, quote=True)
+    else:
+        await message.reply("Usage: \n<code>/cedict your query</code>"
+                            "\n\nFor example:\n<code>/cedict China</code>"
+                            "\n\nYou can also use <code>/dic</code> instead of"
+                            " <code>/cedict</code>",
+                            quote=True)
```

Comment:
I would rewrite it this way

```py
@Client.on_message(filters.command(["cedict", "dict", "dic"])
                   & filters.incoming
                   & ~filters.edited)
async def command_cedict(_, m: Message):
    """/cedict question - get translation from cedict"""
    query = command_args_to_str(m.command)
    if query:
        first_page = 0
        cedict_response = get_cedict_result(query,
                                            first_page,
                                            CEDICT_ENTRIES_PER_PAGE)
        text = cedict_response.formatted_result
        reply_markup = (
            _make_next_page_button() if cedict_response.has_next_page
            else None
        )
    else:
        text = (
            "Usage: \n<code>/cedict your query</code>"
            "\n\nFor example:\n<code>/cedict China</code>"
            "\n\nYou can also use <code>/dic</code> instead of"
            " <code>/cedict</code>"
        )
        reply_markup = None
    await m.reply(text, reply_markup=reply_markup, quote=True)
```

Reply:
```
reply_markup = (
            _make_next_page_button()
            if cedict_response.has_next_page
            else None
        )
```
oh didn't know about this python feature, cool